config/bootstrap.php

```diff
@@ -60,7 +60,7 @@
 use Cake\Error\ErrorHandler;
 use Cake\Log\Log;
 use Cake\Mailer\Email;
-use Cake\Network\Request;
+use Cake\Http\ServerRequest;
```

Comment:
You also need to change usages like `Request::addDetector()` in the file to `ServerRequest::addDetector()`.